src/components/Collapsible/Collapsible.tsx

```diff
@@ -17,12 +17,23 @@ export interface Props {
   children?: React.ReactNode;
 }
 
+enum TransitionStatus {
+  Entering = 'entering',
+  Entered = 'entered',
+  Exiting = 'exiting',
+  Exited = 'exited',
+}
+
 const CSS_VAR_COLLAPSIBLE_HEIGHT = '--polaris-collapsible-height';
 const CSS_VAR_COLLAPSIBLE_TRANSITION_DURATION =
```

Comment:
Since we aren't doing any calculations anymore to adjust speed, might as well put this one in the CSS I think.